### PR TITLE
Blue Ocean: Version 1.000 added

### PR DESCRIPTION
**Pull request by eliheuer**
Note: This is a draft PR for now and should not be merged. 

From the upstream repo: https://github.com/eliheuer/blue-ocean
At commit: https://github.com/eliheuer/blue-ocean/commit/0a58b7a384872ac888dde753bc07d922d0475b67

## Fontbakery report

Fontbakery version: 0.7.28

<details>
<summary><b>[1] Family checks</b></summary>
<details>
<summary>⚠ <b>WARN:</b> Is the command `ftxvalidator` (Apple Font Tool Suite) available?</summary>

* [com.google.fonts/check/ftxvalidator_is_available](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/universal.html#com.google.fonts/check/ftxvalidator_is_available)
<pre>--- Rationale ---

There&#x27;s no reasonable (and legal) way to run the command `ftxvalidator` of the
Apple Font Tool Suite on a non-macOS machine. I.e. on GNU+Linux or Windows etc.

If Font Bakery is not running on an OSX machine, the machine running Font
Bakery could access `ftxvalidator` on OSX, e.g. via ssh or a remote procedure
call (rpc).

There&#x27;s an ssh example implementation at:
https://github.com/googlefonts/fontbakery/blob/master/prebuilt/workarounds
/ftxvalidator/ssh-implementation/ftxvalidator

</pre>

* ⚠ **WARN** Could not find ftxvalidator.

</details>
<br>
</details>
<details>
<summary><b>[5] BlueOcean[wght].ttf</b></summary>
<details>
<summary>⚠ <b>WARN:</b> Checking OS/2 achVendID.</summary>

* [com.google.fonts/check/vendor_id](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/vendor_id)
<pre>--- Rationale ---

Microsoft keeps a list of font vendors and their respective contact info. This
list is updated regularly and is indexed by a 4-char &quot;Vendor ID&quot; which is
stored in the achVendID field of the OS/2 table.

Registering your ID is not mandatory, but it is a good practice since some
applications may display the type designer / type foundry contact info on some
dialog and also because that info will be visible on Microsoft&#x27;s website:

https://docs.microsoft.com/en-us/typography/vendors/

This check verifies whether or not a given font&#x27;s vendor ID is registered in
that list or if it has some of the default values used by the most common font
editors.

Each new FontBakery release includes a cached copy of that list of vendor IDs.
If you registered recently, you&#x27;re safe to ignore warnings emitted by this
check, since your ID will soon be included in one of our upcoming releases.

</pre>

* ⚠ **WARN** OS/2 VendorID value 'NONE' is not yet recognized. If you registered it recently, then it's safe to ignore this warning message. Otherwise, you should set it to your own unique 4 character code, and register it with Microsoft at https://www.microsoft.com/typography/links/vendorlist.aspx
 [code: unknown]

</details>
<details>
<summary>⚠ <b>WARN:</b> Check copyright namerecords match license file.</summary>

* [com.google.fonts/check/name/license](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/name/license)
<pre>--- Rationale ---

A known licensing description must be provided in the NameID 14 (LICENSE
DESCRIPTION) entries of the name table.

The source of truth for this check (to determine which license is in use) is a
file placed side-by-side to your font project including the licensing terms.

Depending on the chosen license, one of the following string snippets is
expected to be found on the NameID 13 (LICENSE DESCRIPTION) entries of the name
table:
- &quot;This Font Software is licensed under the SIL Open Font License, Version 1.1.
This license is available with a FAQ at: https://scripts.sil.org/OFL&quot;
- &quot;Licensed under the Apache License, Version 2.0&quot;
- &quot;Licensed under the Ubuntu Font Licence 1.0.&quot;

Currently accepted licenses are Apache or Open Font License.
For a small set of legacy families the Ubuntu Font License may be acceptable as
well.

When in doubt, please choose OFL for new font projects.

</pre>

* ⚠ **WARN** Please consider using HTTPS URLs at name table entry [plat=3, enc=1, name=13] [code: http-in-description]
* ⚠ **WARN** For now we're still accepting http URLs, but you should consider using https instead.
 [code: http]

</details>
<details>
<summary>⚠ <b>WARN:</b> License URL matches License text on name table?</summary>

* [com.google.fonts/check/name/license_url](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/name/license_url)
<pre>--- Rationale ---

A known license URL must be provided in the NameID 14 (LICENSE INFO URL) entry
of the name table.

The source of truth for this check is the licensing text found on the NameID 13
entry (LICENSE DESCRIPTION).

The string snippets used for detecting licensing terms are:
- &quot;This Font Software is licensed under the SIL Open Font License, Version 1.1.
This license is available with a FAQ at: https://scripts.sil.org/OFL&quot;
- &quot;Licensed under the Apache License, Version 2.0&quot;
- &quot;Licensed under the Ubuntu Font Licence 1.0.&quot;

Currently accepted licenses are Apache or Open Font License.
For a small set of legacy families the Ubuntu Font License may be acceptable as
well.

When in doubt, please choose OFL for new font projects.

</pre>

* ⚠ **WARN** Please consider using HTTPS URLs at name table entry [plat=3, enc=1, name=13] [code: http-in-description]
* ⚠ **WARN** Please consider using HTTPS URLs at name table entry [plat=3, enc=1, name=13] [code: http-in-description]
* ⚠ **WARN** Please consider using HTTPS URLs at name table entry [plat=3, enc=1, name=13] [code: http-in-description]
* ⚠ **WARN** Please consider using HTTPS URLs at name table entry [plat=3, enc=1, name=14] [code: http-in-license-info]
* ⚠ **WARN** For now we're still accepting http URLs, but you should consider using https instead.
 [code: http]

</details>
<details>
<summary>⚠ <b>WARN:</b> METADATA.pb: Fontfamily is listed on Google Fonts API?</summary>

* [com.google.fonts/check/metadata/listed_on_gfonts](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/metadata/listed_on_gfonts)

* ⚠ **WARN** Family not found via Google Fonts API. [code: not-found]

</details>
<details>
<summary>⚠ <b>WARN:</b> Checking Vertical Metric Linegaps.</summary>

* [com.google.fonts/check/linegaps](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/hhea.html#com.google.fonts/check/linegaps)

* ⚠ **WARN** OS/2 sTypoLineGap is not equal to 0. [code: OS/2]

</details>
<br>
</details>

### Summary

| 💔 ERROR | 🔥 FAIL | ⚠ WARN | 💤 SKIP | ℹ INFO | 🍞 PASS | 🔎 DEBUG |
|:-----:|:----:|:----:|:----:|:----:|:----:|:----:|
| 0 | 0 | 6 | 37 | 9 | 123 | 0 |
| 0% | 0% | 3% | 21% | 5% | 70% | 0% |

**Note:** The following loglevels were omitted in this report:
* **SKIP**
* **INFO**
* **PASS**
* **DEBUG**

### CODE REVIEW

**Comment by eliheuer** [comment]
This is a fork of [VTF Bluu Next](https://velvetyne.fr/fonts/bluu/) I'm working on. I started this pull request in draft mode to ask for feedback from a few people(@davelab6, @RosaWagner, @jbmorizot, @velvetyne, @BlackFoundry). 

My forked repository is here: https://github.com/eliheuer/blue-ocean

variable font specimen

I'm interested in using quadratic curves to draw variable fonts, so I redrew Bluu Next in quadratic curves with no on-curve to on-curve line segments. This drawing style gives the typeface a different look, my hope is that this fork is sufficiently different from Bluu Suuperstar that I'm not bothering anyone by working on it. I offer a link to the Bluu Suuperstar page on the Black Foundry site in the readme saying:

>A proprietary fork of Bluu Next was released as Bluu Suuperstar by Black Foundry, if you want a version of this typeface with a more traditional outline style and better spacing, please buy a license for the commercial version here: https://black-foundry.com/fonts/bluu-suuperstar

To comply with the [OFL RFN](https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL_web_fonts_and_RFNs), I changed the name to "Blue Ocean", but I could call it "Bluuuu Ocean", or any other name anyone suggests.

Please see the image below that compares the different forks of Bluu Next, note that I avoid most of the forms introduced in Bluu Suuperstar.

comparison helper image

In summary, if anyone is uncomfortable with this project for technical, ethical, or aesthetic reasons and wants me to stop working on it, please let me know here. There are not many precedents for OFL fonts that have proprietary forks because the [OFL](https://scripts.sil.org/cms/scripts/page.php?item_id=OFL_web): PERMISSION & CONDITIONS § 5 seems to forbid relicensing forks?

If there are no complaints I will continue work in this PR with the goal of adding this variable font to the Google Fonts collection.

**Comment by jbmorizot** [comment]
Hi Eli,
First, I want to thank you for tagging us and ask about Blue Ocean. Having both an OFL version aka "Bluu Next") and a proprietary one (aka "Bluu Suuperstar") is not the most common thing I reckon. Still, Bluu Suuperstar was redrawn from scratch using 2 weights (Regular and Black) that don't exist in the OFL version, so I don't know if we can call it a fork. The design ideas come from Bluu Next, but not the outlines.

About Blue Ocean, to my mind some design decisions come from Bluu Suuperstar, like the more traditional width of the "a" (Bluu Next has a quite wide "a"), or the more quiet axis of the "c", "e" and "o". If you look at your Regular next to Bluu Suuperstar Regular, you can see you followed a little too much our redesign (not on purpose, I'm not accusing you of anything like that). You Black is clearly more different and has a voice on its own. I think it's the way to follow.

Regarding the ethical side: it's obvious the quite wide audience of your publisher will impact both Velvetyne and Black[Foundry] websites. I have some concern about the life and sales cycle of Bluu Suuperstar, when a multi-width version is available freely with one CSS copy-paste. Bluu Suuperstar is part of the Black[Foundry] catalog and so of its design identity and its turnover. Therefore even if the lack of discrete instances and italics make a clear difference, Blue Ocean will probably be a game changer. But it's a little late to ask you to delete all your efforts at this point since a V1 is pushed, so I feel I don't really have a choice at this stage.

**Comment by eliheuer** [comment]
Thanks, your feedback is exactly what I was looking for. Making progress on that project is a lot of work, and I have lots of other projects I could be working on. So if you or anyone else involved with the history of that typeface is not happy with the direction I'm going, then I don't feel like it's worth the effort to continue.

If I do end up doing any more work on this, I would need to spend lots of time making it look less like Bluu Suuperstar, and would update this PR with a new demo image for review.

But for now, I'm on to other fonts that need work. Thanks again!

I think the only way the OFL as an ecosystem is going to work is if there is trust and understanding between the different groups that use it. So as far as I'm concerned moving forward with this project in its current state is not an option.

**Comment by eliheuer** [comment]
I made my forked repository private, so no one else will continue work on this.
